src/main/java/com/contrastsecurity/sdk/ContrastSDK.java

```diff
@@ -761,6 +761,35 @@ public Traces getTraces(String organizationId, String appId, TraceFilterForm for
     }
   }
 
+  /**
```

Comment:
If we reuse the `TraceFilterForm` we can also just update the existing `getTraces` method to use V2 vulnerability filtering. Both methods are getting a list of vulnerabilities based on a filter.